system-addon/lib/SnippetsFeed.jsm

```diff
@@ -138,7 +141,7 @@ this.SnippetsFeed = class SnippetsFeed {
         this.uninit();
         break;
       case at.SHOW_FIREFOX_ACCOUNTS:
-        this.showFirefoxAccounts(action._target.browser);
+        await this.showFirefoxAccounts(action._target.browser);
```

Comment:
Don't `await` here,  just calling`showFirefoxAccounts`is sufficient

Reply:
I actually need to await on that for testing reasons :/

Reply:
Your stub doesn't have to return a `Promise`. Just do `return url;` and test without the await.

Reply:
This will not work, `await` does not behave in a synchronous way with non-promises.  

From [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await): 
> If the value of the expression following the await operator is not a Promise, it's converted to a resolved Promise.